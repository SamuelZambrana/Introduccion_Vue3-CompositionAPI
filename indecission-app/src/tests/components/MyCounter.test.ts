import { test, describe, expect } from "vitest";
import { mount } from "@vue/test-utils";
import  MyCounter  from "@/components/MuyCounter.vue"

describe('MyCounter Component', () => {

    test('should be tested', () => {
        
        const wrapper = mount(MyCounter, {
            props: { 
                value: 10, 
            }
        });
        expect(wrapper.html()).toMatchSnapshot();

    });
});