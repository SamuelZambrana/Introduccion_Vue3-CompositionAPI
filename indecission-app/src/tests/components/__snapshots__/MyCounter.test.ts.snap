// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MyCounter Component > should be tested 1`] = `
"<section>
  <h3>counter: 5</h3>
  <h3>Square: 25</h3>
  <div><button>+1</button><button>-1</button></div>
</section>"
`;
